src/components/index-page/use-cases/index.tsx

```diff
@@ -0,0 +1,213 @@
+"use client"
+
+import clsx from "clsx"
+import { useState, Fragment, ReactNode } from "react"
+
+import { Button } from "@/app/conf/_design-system/button"
+import { StripesDecoration } from "@/app/conf/_design-system/stripes-decoration"
+import ArrowDownIcon from "@/app/conf/_design-system/pixelarticons/arrow-down.svg?svgr"
+
+import blurBean from "./blur-bean.webp"
+
+type UseCase = {
+  label: string
+  description: ReactNode
+  cta: string
+  href: string
+}
+
+const USE_CASES: UseCase[] = [
+  {
+    label: "A large backend with many services",
+    description:
+      "GraphQL serves as a unified data layer across multiple services. This way you simplify API management and reduce dependencies between teams. It enables efficient data fetching while keeping the API surface flexible and maintainable.",
+    cta: "Best Practices for Large-Scale Systems",
+    href: "/learn/best-practices",
+  },
+  {
+    label: "A mobile app",
+    description: (
+      <>
+        GraphQL lets you request exactly what you need in one call with no
+        overfetching to preserve battery and work on slow networks. With
+        libraries like{" "}
+        <a
+          className="typography-link"
+          href="https://nearform.com/open-source/urql/docs/graphcache/offline/#offline-behavior"
+        >
+          GraphCache
+        </a>{" "}
+        your app can work offline on planes and trains, and versionless schema
```

Comment:
Let's keep the landing page vendor neutral? 

```suggestion
        overfetching to preserve battery and work on slow networks. With
        a persistent normalized cache,
        your app can work offline on planes and trains, and versionless schema
```